static/app/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/constants.tsx

```diff
@@ -1,10 +1,8 @@
 export const BUBBLE_SERIES_ID = '__release_bubble__';
 export const BUBBLE_AREA_SERIES_ID = '__release_bubble_area__';
-export const DEFAULT_BUBBLE_SIZE = 12;
+export const DEFAULT_BUBBLE_SIZE = 4;
 
 // "padding" around the bubbles as it is drawn on canvas vs CSS, this may need
 // to move into `renderReleaseBubble` if it needs to be customizable
-export const RELEASE_BUBBLE_Y_PADDING = 8;
-export const RELEASE_BUBBLE_Y_HALF_PADDING = RELEASE_BUBBLE_Y_PADDING / 2;
+export const RELEASE_BUBBLE_Y_PADDING = 2;
```

Comment:
tbh we probably don't even need these, could move them as default args in the hook